### PR TITLE
Ionut/dashboard page

### PR DESCRIPTION
**Pull request by bodogaii**
As we don't have yet imported the core logic and I`m not sure how the wallet object will look, I extracted some variables from the render function and named them by "eye" which will by modified when we will know for sure and even the method how they will be injected in the dashboard component. I think that the copyToClipboard function should be extracted in a Util class but I let it in the dashboard class for now for a later decision. I want to mention that the Copy Icon doesn't ripple when clicked, I hope I will recieve a suggestion or a little help about this.

### CODE REVIEW

File: src/app/pages/dashboard/dashboard.tsx [hunk 1]
```diff
@@ -1,10 +1,71 @@
-import { h, Component } from 'preact';
+import {h, Component} from 'preact';
+
+import {TopBar} from "../layouts/top-bar/top-bar";
+import LayoutGrid from "preact-material-components/LayoutGrid";
+import Card, {CardMediaContent} from "preact-material-components/Card";
+import 'preact-material-components/Card/style.css';
```
Comment (cristianbote) [inline src/app/pages/dashboard/dashboard.tsx]: We should be consistent with the quotation marks. Seems like we should use the double one.

File: src/app/pages/dashboard/dashboard.tsx [hunk 2]
```diff
@@ -1,10 +1,71 @@
...
 export class DashboardPage extends Component {
+
+    private wallet: any;
+
+    constructor() {
+        super();
+        this.wallet = {coin: "ZIL", address: "0x5FC7409B4B41E06E73BA1AA7F3127D93C76BD557", balance: "0.00000000"};
+    }
+
+    //Extract this to a util class
```
Comment (cristianbote) [inline src/app/pages/dashboard/dashboard.tsx]: We should use `jsdoc` syntax for documenting the code.

File: src/app/pages/layouts/top-bar/top-bar.tsx [hunk 3]
```diff
@@ -10,8 +11,13 @@ export class TopBar extends Component {
                     <TopAppBar.Section align-start>
                         <img class="top-appbar-icon" src="/assets/logo.svg"></img>
                     </TopAppBar.Section>
+                    <TopAppBar.Section>
```
Comment (cristianbote) [inline src/app/pages/layouts/top-bar/top-bar.tsx]: Using JSX does not equals html output. So, we should _always_ use self-closing tags, if there's no content inside.
```js
<FooComponent baz={false}></FooComponent>

// instead use this:
<FooComponent baz={false} />
```

File: package.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
     "build": "npm run build:web -- --dest build/web",
     "build:web": "preact build --config ./config/web.config.js --template src/web/index.html --no-prerender",
     "build:ext": "preact build --config ./config/extension.config.js --template src/extension/index.html --no-prerender",
-    
+
```
Comment (cristianbote) [inline package.json]: Newlines inside a `json` structure is not spec compliant.

File: src/app/pages/dashboard/dashboard.tsx (hunk 1 above)
Comment (bodogaii) [reply src/app/pages/dashboard/dashboard.tsx]: Done

File: src/app/pages/dashboard/dashboard.tsx [hunk 5]
```diff
@@ -42,23 +42,23 @@ export class DashboardPage extends Component {
...
+                                <p className="mdc-typography--headline5">{this.state.coin} {this.state.balance}</p>
                             </CardMediaContent>
                         </Card>
                     </LayoutGrid.Cell>
                     <LayoutGrid.Cell cols={12}>
                         <Card className="address-card">
                             <CardMediaContent className="address-card-text">
                                 <p className="mdc-typography--body2">Wallet address</p>
-                                <p className="mdc-typography--headline5" id="walletAddress">{this.wallet.address}</p>
+                                <p className="mdc-typography--headline5" id="walletAddress">{this.state.address}</p>
```
Comment (krisboit) [inline src/app/pages/dashboard/dashboard.tsx]: let's try to convert this paragraph in a textarea and use it to copy to clipboard, so no more DOM append and remove needed ;) 

Also for selection of an element from DOM you can use `ref` property. See an example here: https://jsfiddle.net/developit/x8qpzjdf/

Comment (bodogaii) [reply src/app/pages/dashboard/dashboard.tsx]: Ty. I'l take a look now. Also I will delete the bottom bar classes as you suggested and make some squashes for the commits to keep things cleaner

**Pull request merged**